envs/quadrotors/quad_multi_model.py

```diff
@@ -11,11 +11,15 @@ class QuadMultiMeanEncoder(EncoderBase):
     def __init__(self, cfg, obs_space, timing, self_obs_dim=18, neighbor_obs_dim=6, neighbor_hidden_size=32, obstacle_obs_dim=6, obstacle_hidden_size=32):
         super().__init__(cfg, timing)
         self.self_obs_dim = self_obs_dim
-        self.neighbor_obs_dim = neighbor_obs_dim
+        if cfg.obs_space == 'extend+': self.neighbor_obs_dim = 9  # include goal pos info
+        else: self.neighbor_obs_dim = neighbor_obs_dim
```

Comment:
Do we actually ever pass non-default neighbor_obs_dim parameter to this function?
I think it only causes extra confusion now.

We should just set self.neighbor_obs_size based on `cfg.obs_space`

Reply:
Default is meant to be equivalent to extend_obs=False from before